companion-app/src/Components/UploadFile.js

```diff
@@ -140,7 +169,14 @@ class UploadFile extends Component {
         packet.parameters.echo.feedback = this.state.echofeedback;
         packet.parameters.echo.wet = this.state.echowet;
         packet.parameters.echo.dry = this.state.echodry;
-        
+        packet.parameters.flange.apply = this.state.flangeapply;
+        packet.parameters.flange.mix = this.state.flangemix;
+        packet.parameters.flange.depth = this.state.flangedepth;
+        packet.parameters.flange.rate = this.state.flangerate;
+        packet.parameters.pitch.apply = this.state.pitchapply;
+        packet.parameters.pitch.pitch = this.state.pitchpitch;
+        packet.parameters.pitch.fft = this.state.pitchfft;
+        packet.parameters.pitch.channels = this.state.pitchchannels;
```

Comment:
The object in the javascript object's field is named "pitchshift" rather than "pitch" as is written here. This produces and error when trying to send the packet in the web app.